src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -208,17 +209,86 @@
 
   describe 'GET #new' do
     before do
-      login(user_moi)
+      login user_moi
       get :new, name: 'ProjectName'
     end
 
     it { expect(assigns(:project)).to be_a(Project) }
     it { expect(assigns(:project).name).to eq('ProjectName') }
   end
 
+  describe 'GET #show' do
+    before do
+      Project.any_instance.stubs(:number_of_build_problems).returns(0) # To not ask backend for build status
```

Comment:
This is only called at `load_project_info` before filter on `ProjectController`, and this before filter is not that widely used:
```ruby
  before_filter :load_project_info, :only => [:show, :packages_simple, :rebuild_time,
                                              :maintained_projects, :add_maintained_project_dialog,
                                              :add_maintained_project, :remove_maintained_project]
```
I don't know if it worth...